opba-protocols/protocol-bpmn-shared/src/main/java/de/adorsys/opba/protocol/bpmnshared/service/eventbus/ProcessResultEventHandler.java

```diff
@@ -58,7 +61,7 @@ public void handleEvent(InternalProcessResult result) {
         synchronized (lock) {
             InternalProcessResult handledResult = result;
 
-            if (handledResult instanceof ProcessError) {
+            if (handledResult instanceof ProcessError && !(handledResult instanceof InternalReturnableProcessError)) {
```

Comment:
Why
`&& !(handledResult instanceof InternalReturnableProcessError)` 
is needed?
`InternalReturnableProcessError` should have the same logic as `ProcessError` has here

Reply:
Done (by your patch)